platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/merkle/VirtualRootNode.java

```diff
@@ -965,7 +965,7 @@ public void onShutdown(final boolean immediately) {
         if (dataSource != null) {
             try {
                 dataSource.close();
-            } catch (final IOException e) {
+            } catch (final Throwable e) {
```

Comment:
Here as well.

Reply:
Will change it to `Exception`

Reply:
This method is expected to throw IOException, so if that's caught it should rethrow.
Might be best to split as IOException and RuntimeException so both can be rethrown...